flow-typed/js-cookie.js

```diff
@@ -1,5 +1,6 @@
 declare module 'js-cookie' {
   declare export default {
+    get(name: string): ?string,
```

Comment:
do we really need this?

Reply:
Yes, otherwise Flow complains of a missing property:

> Cannot call cookies.get because property get is missing in object type

I only noticed it now, because the file `test/config.js` didn't contain the `// @flow` annotation and was not being checked by Flow.